src/Compiler/Microsoft.NET.Sdk.Razor.SourceGenerators/StaticCompilationTagHelperFeature.cs

```diff
@@ -38,12 +38,10 @@ public List<TagHelperDescriptor> GetDescriptors()
             return results;
         }
 
-        IReadOnlyList<TagHelperDescriptor> ITagHelperFeature.GetDescriptors() => GetDescriptors();
+        IReadOnlyList<TagHelperDescriptor> ITagHelperFeature.GetDescriptors() => GetDescriptors(targetSymbol: null);
 
         public Compilation? Compilation { get; set; }
```

Comment:
Seems that `Compilation` is set only during initialization, could we make it `init`-only? And maybe `required` and non-null? Would also remove the static `EmptyList` - that looks very dangerous.